style/post.css

```diff
@@ -60,8 +60,7 @@
   font-size: 1rem;
   line-height: 1.5;
   padding: 0 32px;
-  box-sizing: content-box;
```

Comment:
the image caption will inherit the `box-sizing: border-box` property defined in https://github.com/adobe/theblog/blob/master/style.css#L18-L23